README.md

```diff
@@ -171,13 +172,13 @@ For more information on configuring Pa11y, see the [configuration documentation]
 The ignore flag can be used in several different ways. Separated by semi-colons:
 
 ```
```

Comment:
is there a reason for changing this to issue-code-_n_? If I were browsing the docs right now looking for info on this I'd be like "wait, what's issue-code-1? how do i find that out?" as it's not really obviously documented. It's described in the results object below, but you'd need to read down to a totally different section to figure it out

Reply:
I mostly wanted to switch away from warning/notice because they're ignored by default and there's no point adding them here. What would you suggest as an alternative? Use real issue codes?

Reply:
honestly i'd think about removing the ignore flag from the CLI entirely. 

As it is, people would need to switch on non-errors and then also potentially ignore some error code in the same shell command. seems like a bit of a weird and confusing thing to do and even documenting it will be quite confusing :/

Reply:
i am sick though, so maybe it's just that my brain is failing to work properly

Reply:
Hmm, I'm not sure how I feel about removing it if we _can_ support it because of how it would break. I wanted to keep the CLI with as few a changes as possible (it's the most used thing) and this one would just exit a Pa11y process immediately if an `--ignore` flag was present.

Reply:
I think it's the circularity of it that's bugging me. Default to don't show this, but show this, but don't show that other thing that's part of this. 

What if there was an explicit `--show warning --show notice` (or `--show-error-code-_n_`, i suppose, but I still think that _n_ needs to be explicitly documented at this stage) instead of `--ignore`? Would that cause too many problems with backwards compatibility?

Reply:
I guess something like `--include-warnings` and `--include-notices` would be alright, so basically just split the `--show-non-errors` in two?

Reply:
yeah, i think that would be clearer. From a user perspective that would just reverse the 4.x settings which seems easier to understand

Reply:
Done 👍